src/furtherWork/petG.js

```diff
@@ -0,0 +1,70 @@
+// GAME STYLE PET
+const {
+    grootTypes,
+    battleCries,
+    attackMoves
+} = require("./gameProperties.js");
+
+
+POTION_HEAL = 10;
+
+
+class Groot {
+    constructor(name, moves=[]) {
+        this.name = name
+        this.battlecry = battleCries[Math.floor(Math.random() * battleCries.length)];
+        this.moves = moves;
+        this.species = grootTypes[Math.floor(Math.random() * grootTypes.length)];
+        this.weakness = this.weakness;
+        this.health = 100;
+       
+    }
+
+get isAlive() {
+    return this.health > 0;
+}
+
+battleCry(groot) {
+    this.health += 5
+    return `${this.name} shouts ${this.battlecry}`
+}
+
+/*attack(target, move) {
+const hasMove = this.moves.includes(move)
+if (hasMove) {
+    target.health -= move.damage;
+    return `${target.name} health reduced to ${target.health}`;
+} else {
+    return `${this.name} doesn't have that move!`;
+}
+} */
```

Comment:
Why is this commented out? The test associated with it fails when uncommenting it, if its not completed yet some explanation about what state its in should be given in the readme and the failing test skipped until you have time to finish it off. As it stands anyone new to the project would see this and have no idea why its commented out or why the test is failing, so its good to give a bit more info about whats going on in cases like this